.gitignore

```diff
@@ -10,3 +10,4 @@ tags
 tegra/
 *.i
 .download*
+.ccls-cache
```

Comment:
Please revert change in .gitignore file.

A general recommendation about IDE-specific files is using of global .gitignore approach: https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Fixed